### ISSUE TITLE
The asset manager may display that a job is "Queued" when in fact the job is "Blocked"

### ISSUE THREAD

**Issue by tst-ccamp** [google/earthenterprise#221]
#216 and #182 both provide examples of instances where a job reported is "Queued" is revealed as being blocked when investigating the output from getop. However, even getop may still incorrectly display the status of the job as "Waiting", even though the job is blocked by an external issue. 

For example:
```
Waiting:
  res/alot.ktasset/product.kta?version=1
     Not enough disk space to make reservation for output files.Task res/alot.ktasset/product.kta?version=1...
```

I don't believe freeing up enough disk space will cause the job to proceed without restarting the fusion server. This is something that should also be investigated, perhaps in a follow-up issue.
